src/mozanalysis/metrics/__init__.py

```diff
@@ -206,14 +242,6 @@ class Metric:
     description = attr.ib(type=Optional[str], default=None)
     bigger_is_better = attr.ib(type=bool, default=True)
 
-    def from_data_source(self, data_source: DataSource) -> "Metric":
```

Comment:
I think I agree with deleting this. On the one hand we may want to use this for glean apps on different platforms. On the other hand, copypasta in this context is probably safer because a Fenix data scientist could unknowingly break the iOS metrics.

Reply:
Yeah I thiiiiiiiiink it just works out to be an attractive nuisance but it's easy enough to revert if we want it back.